### ISSUE TITLE
How many memory do you need for waymo, I downloaded the data set and find it huge as 800g, btw, how long did you train with waymo and how many 2080tis you used

### ISSUE THREAD

**Issue by yanghu819** [yifanzhang713/IA-SSD#11]

**Comment by yifanzhang713** [comment]
From my experience, the Waymo dataset is more than 1 TB including 800 GB training set and 200GB validation set. And you need at least another block of storage of the same size to prepare the dataset for OpenPCDet. 

We train our model on 8 RTX 2080Ti GPUs and it takes around 12 hours on 20% data (~32k frames) of all the training samples.

**Comment by yanghu819** [comment]
Thank you！Amazing work!
